### DISCUSSION TITLE
DQD Tool Installation

### DISCUSSION THREAD

**Discussion by ranatech08** [OHDSI/DataQualityDashboard#343] [General]
Hi,

I am new to R and also to your DQD tool.

Can I have the link for complete documentation of DQD setup? I don't seem to find it here in the repo. 
For instance, I see the script: codeToRun.R under extras, wherein I "think" the DB details need to be inserted. Please confirm?

If yes, then I need to change the libpath for the install_github() for me to trace that script file and make the required changes. Will be great if instruction like these be documented with the users for easier setup and utilization. 

And if not, then which file do I need to make the changes. Overall, a general documentation for DQD setup is required, which would be really great!!!

Alternatively, a YouTube video on this will be fabulous. 

Thanks,
Rana

**Comment by don-torok** [comment]
The forums are a better place to ask questions.  None the less, there should be an git enhancement issue to add installation instructions or a  link to them in the Readme.  [Installation instructions](https://ohdsi.github.io/DataQualityDashboard/articles/DataQualityDashboard.html).  I suggest you google "OHDSI data quality dashboard" for additional information.

**Comment by katy-sadowski** [comment]
Installation instructions are now included in README, and package website is also linked in README. We also have an open issue for a more detailed step-by-step setup guide (#316 )
